mysql-test/mariadb-test-run.pl

```diff
@@ -2266,6 +2266,14 @@ sub environment_setup {
 		   "$path_client_bindir/my_print_defaults");
   $ENV{'MYSQL_MY_PRINT_DEFAULTS'}= native_path($exe_my_print_defaults);
 
+  # ----------------------------------------------------
+  # mariadb-upgrade-config-file
+  # ----------------------------------------------------
+  my $exe_mariadb_upgrade_config_file=
+    mtr_exe_exists("$bindir/extra$multiconfig/mariadb-upgrade-config-file",
+		   "$path_client_bindir/mariadb-upgrade-config-file");
+  $ENV{'MARIADB_UPGRADE_CONFIG_FILE'}= native_path($exe_mariadb_upgrade_config_file);
```

Comment:
If you require an executable to be built for mtr, you must update the list in `ADD_DEPENDENCIES(minbuild` in the top level CMakeLists.txt with your executable.  minbuild is the minimal target that is MTR-capable, and it is used in the appveyor build

Reply:
Oh, I see, it is already done, so ignore the above